models/openacademy_course.py

```diff
@@ -8,3 +8,13 @@ class Course(models.Model):
     description = fields.Text()
     responsible_id = fields.Many2one("res.users", ondelete="restrict")
     session_ids = fields.One2many("openacademy.session", string="Sessions", inverse_name="course_id")
+
+    _sql_constraints = [("description_and_title_different", "CHECK(name != description)", "The title and description of the course must be different."), ("unique_course_name", "UNIQUE(name)", "The name of the course must be unique.")]
```

Comment:
Please indent, this line is too long (it should be at most 119 characters). Doesn't vim mark this as a lint?